gen/generate.go

```diff
@@ -326,24 +340,25 @@ func generateGenesisFiles(outDir string, protoVersion protocol.ConsensusVersion,
 		fmt.Fprintln(verboseOut, protoVersion, protoParams.MinBalance)
 	}
 
+	if rewardsBalance < protoParams.MinBalance {
+		// Needs to at least have min balance
+		rewardsBalance = protoParams.MinBalance
+	}
+
 	records["FeeSink"] = basics.AccountData{
 		Status:     basics.NotParticipating,
 		MicroAlgos: basics.MicroAlgos{Raw: protoParams.MinBalance},
 	}
 	records["RewardsPool"] = basics.AccountData{
 		Status:     basics.NotParticipating,
-		MicroAlgos: basics.MicroAlgos{Raw: defaultIncentivePoolBalanceAtInception},
+		MicroAlgos: basics.MicroAlgos{Raw: rewardsBalance},
 	}
 
 	sinkAcct := genesisAllocation{
-		Name:   "FeeSink",
-		Stake:  protoParams.MinBalance,
```

Comment:
Opening a thread because the change feels sensitive and I want to confirm other reviewers consider it.

The delete here and below for `poolAcct` are based on analysis in https://github.com/barnjamin/go-algorand/pull/1.  I'd appreciate a sanity check to confirm I didn't miss an intended usage.

Reply:
I see my mistake. These objects are strictly used to lookup the name key as an index into the `records` map which has duplicate definitions just above.